packages/block-scripts/.eslintrc.cjs

```diff
@@ -3,6 +3,8 @@ module.exports = {
   ...require("@local/eslint-config/generate-workspace-config.cjs")(__dirname),
   rules: {
     "no-console": "off",
-    "import/extensions": ["error", "always"],
+    "import/extensions": ["error", "ignorePackages"],
```

Comment:
Allows us to remove `// eslint-disable-next-line` before `import { fileFromPath } from "formdata-node/file-from-path"`. This still works because `formdata-node` is CJS and is therefore subject to a ‘classic’ file resolution mechanism.